data/js/tomahawk.js

```diff
@@ -494,6 +494,7 @@ Tomahawk.ajax = function(url, settings) {
 
     settings.type = settings.type || settings.method || 'get';
     settings.method = settings.type;
+    settings.encoding = settings.encoding || 'form';
```

Comment:
Not sure `encoding` is very optimal. It might also refer to the charset... on the other hand anything one sends accross the wire nowadays should be utf8 anyway ...